tools/ci_connector_ops/setup.py

```diff
@@ -2,39 +2,62 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
+from pathlib import Path
+
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
```

Comment:
same as above, i think we should promote this to a top level .env file (even at repo level!)